help/destinations/catalog/crm/salesforce.md

```diff
@@ -75,12 +75,16 @@ Note down the items below before you authenticate to the Salesforce CRM destinat
 | Credential | Description | Example |
 | --- | --- | --- |
 | <ul><li>Salesforce domain prefix</li></ul> | See [Salesforce domain prefix](https://help.salesforce.com/s/articleView?id=sf.domain_name_setting_login_policy.htm&type=5) for additional guidance. | <ul><li>If your domain is as below, you need the highlighted value.<br> <i>`d5i000000isb4eak-dev-ed`.my.salesforce.com</i></li></ul>|
-|<ul><li>Consumer Key</li><li>Consumer Secret</li></ul> | Refer to the [Salesforce documentation](https://help.salesforce.com/s/articleView?id=sf.connected_app_rotate_consumer_details.htm&type=5) if you need additional guidance. | <ul><li>r23kxxxxxxxx0z05xxxxxx</li><li>ipxxxxxxxxxxT4xxxxxxxxxx</li></ul> |
+|<ul><li>Consumer Key</li><li>Consumer Secret</li></ul> | Refer to the [Salesforce documentation](https://help.salesforce.com/s/articleView?id=sf.connected_app_rotate_consumer_details.htm&type=5) if you need additional guidance. | <ul><li><code>r23kxxxxxxxx0z05xxxxxx</code></li><li><code>ipxxxxxxxxxxT4xxxxxxxxxx</code></li></ul> |
 
 ### Guardrails {#guardrails}
 
 Salesforce balances transaction loads by imposing request, rate and timeout limits. Refer to the [API Request Limits and Allocations](https://developer.salesforce.com/docs/atlas.en-us.salesforce_app_limits_cheatsheet.meta/salesforce_app_limits_cheatsheet/salesforce_app_limits_platform_api.htm) for details.
 
+> [!IMPORTANT]
+> 
+> Ensure that you populate either *Contact* or *Lead* objects and not both within a Platform segment.
```

Comment:
Please rephrase. What are we trying to convey in this sentence? It is not at all clear and it is not grammatically correct.